### ISSUE TITLE
Determine the general approach for WASM to JS binding generation + demonstrate with a simple function export

### ISSUE THREAD

**Issue by ianthetechie** [stadiamaps/ferrostar#101]
- [x] Task 1: determine what we will use for binding generation. I mentioned on our first intro call that [wasm-bindgen](https://github.com/rustwasm/wasm-bindgen) seems the likely candidate, but feel free to do some research on alternatives.
- [x] Task 2: demonstrate that it works with a simple function export and a test page using Javascript to call the WASM function. (Note that this test code doesn't need to be merged into the project, but we do need to have a quick demo + code to verify that the approach works).

**Comment by ianthetechie** [comment]
I would suggest the following as a path forward once #120 is merged.

First, build a minimal POC: Add an extremely simple function (add two numbers?) to `lib.rs` and export it with wasm-bindgen. Then build a very simple webpage that loads the module and calls the function with JS and does a console log or something. It does not have to be pretty; just demonstrate that it's working.

I would suggest using a conditional dependency and attributes as shown at the end of this page: https://rustwasm.github.io/docs/wasm-bindgen/reference/rust-targets.html. This way we keep binary size smaller on other platforms.

From there, I would suggest that `OsrmResponseParser` is the easiest "real" thing to start with. It is very easy to test, since the core function is parsing string input and parsing it into a `Route`. I'll create a task for that.
